### PR TITLE
[SQUQASH-ME] qa/tasks/vstart_runner.py: s/StringIO/BytesIO/

### PR DESCRIPTION
**Pull request by tchaikov**
as expected by "qa/tasks/cephfs: get rid of StringIO for py3"

Signed-off-by: Kefu Chai <kchai@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
should address the failure of 

```
Traceback (most recent call last):
  File "/home/teuthworker/src/git.ceph.com_git_teuthology_master/teuthology/run_tasks.py", line 89, in run_tasks
    manager.__enter__()
  File "/usr/lib/python2.7/contextlib.py", line 17, in __enter__
    return self.gen.next()
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-03-01-1346/qa/tasks/ceph_objectstore_tool.py", line 197, in task
    ERRORS += test_objectstore(ctx, config, cli_remote, REP_POOL, REP_NAME)
  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-kefu-testing-2020-03-01-1346/qa/tasks/ceph_objectstore_tool.py", line 429, in test_objectstore
    keys = remote.sh(cmd, wait=True, stderr=BytesIO()).split()
NameError: global name 'BytesIO' is not defined
```

http://pulpito.ceph.com/kchai-2020-03-01_09:33:43-rados-wip-kefu-testing-2020-03-01-1346-distro-basic-smithi/4816112/

File: qa/tasks/ceph_objectstore_tool.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 """
 ceph_objectstore_tool - Simple test of ceph-objectstore-tool utility
 """
+from io import BytesIO
```
Comment (kshtsk) [inline qa/tasks/ceph_objectstore_tool.py]: Are you sure you've added this to the right file?

Comment (tchaikov) [reply qa/tasks/ceph_objectstore_tool.py]: @kshtsk please search in this file for `BytesIO`. without this change, we have http://pulpito.ceph.com/kchai-2020-03-02_05:23:09-rados-wip-kefu-testing-2020-03-01-2348-distro-basic-smithi/4818687/

Comment (kshtsk) [reply qa/tasks/ceph_objectstore_tool.py]: ah, right, forgot you based on my branch, sorry :-)

**Comment by kshtsk** [comment]
Thanks, I've cherry-picked commits and squashed your changes to my branch after rebase to the master's tip.

**Pull request closed without merging**